hibernate-jipijapa/src/test/bundles/overridenpar/META-INF/persistence.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Hibernate, Relational Persistence for Idiomatic Java
+  ~
+  ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+  ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Do we need all these additional integrations tests? I wonder if most aren't redundant with the ORM standard tests.

In other words: are they all specific to JipiJapa?

Reply:
Hi Sanne, yes, I think we need the tests to ensure that basic deployment scanning works.

Reply:
We need *so many* just to test scanning/discovery?

Reply:
I forget exactly why we needed so many.  I think the initial idea was to go from zero tests in hibernate-jipijapa, to many of the same tests that are in Hibernate ORM.  Having a smaller number of tests sounds fine to me.